how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -20,33 +18,28 @@ The Kubernetes cluster will use the
 [CRI containerd plugin](https://github.com/containerd/cri) and
 [Kata Containers](https://katacontainers.io) to launch untrusted workloads.
 
+For Kata Containers 1.5.0-rc2 and above, we will use [`containerd-shim-kata-v2` (short as `shimv2` in this documentation)](../design/shimv2.md) 
+launch Kata Containers. While for the previous version of Kata Containers, the Pods will be launched with `kata-runtime`.
+
 ## Requirements
 
 - Kubernetes, kubelet, kubeadm
-- cri-containerd
+- containerd with `cri` plug-in
 - Kata Containers
 
 > **Note:** For information about the supported versions of these components,
 > see the  Kata Containers
 > [versions.yaml](https://github.com/kata-containers/runtime/blob/master/versions.yaml)
 > file.
 
-## Install containerd with CRI plugin enabled
-
-- Follow the instructions from the
-  [CRI installation guide](http://github.com/containerd/cri/blob/master/docs/installation.md).
-
-- Check if `containerd` is now available
-  ```bash
-  $ command -v containerd
-  ```
+## Install and configure containerd
 
-## Install Kata Containers
+Firstly, follow the [How to use Kata Containers and Containerd](containerd-kata.md) to install
```

Comment:
First, follow the [How to use Kata Containers and Containerd](containerd-kata.md) to install and configure containerd. Then, make sure the containerd works with the [examples in it](containerd-kata.md#run).